typing/printtyp.ml

```diff
@@ -479,7 +479,7 @@ and raw_type_desc ppf = function
         row.row_fields
         "row_more=" raw_type row.row_more
         "row_closed=" row.row_closed
-        "row_fixed=" row.row_fixed
+        "row_fixed=" (Btype.is_fixed row)
```

Comment:
That seems like a mistake. When I want to print the raw type, I want all the information in there.

At the very least, I'd want the information displayed to have the right type (so I would accept to see `Some _`, but not `true`).

Reply:
I have upgraded the printer to print the full information.